src/lib/containers/home_page/goals/Goals.tsx

```diff
@@ -144,6 +144,7 @@ export default function (props: GoalsProps) {
           summary,
           link,
           asset,
+          token,
```

Comment:
This is the main bug fix, the rest is to convert old component to use react hooks instead.

Reply:
Got it! Thanks.